src/test/java/utils/ProjectConstants.java

```diff
@@ -13,4 +13,6 @@ public class ProjectConstants {
     public static final String FORMS_URL = BASE_URL + "forms";
     public static final String PRACTICE_FORMS_HEADER = "Practice Form";
     public static final String PRACTICE_FORMS_URL = BASE_URL + "automation-practice-form";
+    public static final String SALT_UPPER_AND_LOWER_LETTERS = "AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz";
```

Comment:
please not hardcode change to regex . for example /^[a-z]+$ (it contains upper and lower cases of alphabet)

Reply:
https://regex101.com/ 
you can verify it here